server/api/offline_batch_results.py

```diff
@@ -185,6 +185,89 @@ def record_offline_batch_results(
     return jsonify(status="ok")
 
 
+@api.route(
+    "/election/<election_id>/jurisdiction/<jurisdiction_id>/round/<round_id>/results/batch/",
+    methods=["POST"],
+)
+@restrict_access([UserType.JURISDICTION_ADMIN])
+def record_offline_batch_results_add(
+    election: Election,  # pylint: disable=unused-argument
+    jurisdiction: Jurisdiction,
+    round: Round,
+):
+    offline_batch_results = request.get_json()
+    validate_offline_batch_results(election, jurisdiction, round, offline_batch_results)
+
+    for batch_results in offline_batch_results:
```

Comment:
shouldn't this endpoint just take one batch at a time?

Reply:
it should, but I was trying to keep it simple so I could reuse your validation. I'll look at simplifying.